### PR TITLE
Added Error Message

### PR DESCRIPTION
**Pull request by john-baute**
Added error message for when submitting report with incomplete parameters.

### CODE REVIEW

**Review by waffy1901 — CHANGES_REQUESTED** [review]
A few things:
- There is no error message for manual reports.
- An error message should pop up if no attributes are selected.
- When multiple parameters are entered - the incomplete fields should be stated in the error message. For instance, in an automatic report, if the cdc csv file and query year are not selected, the error message should display something along the lines of "cdc csv file was not uploaded and a query year was not selected".

**Comment by john-baute** [comment]
There's code for the manual report on lines 105-115, where it builds the string based on what was missing, will implement the changes mentioned now

**Comment by waffy1901** [comment]
> There's code for the manual report on lines 105-115, where it builds the string based on what was missing, will implement the changes mentioned now

I believe you forgot to add setShowError(true) in that code block, which is why the error popup is not displaying.

**Comment by john-baute** [comment]
Ah yeah that was it, good catch

File: CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx [hunk 1]
```diff
@@ -99,6 +103,15 @@ export default function CreateReport({ onDone }) {
...
       if (stateFile === null || cdcFile === null) {
         console.error("Files not uploaded!")
+        var errors = [];
+        if (stateFile === null) {
+          errors.push("State file not uploaded");
+        }
+        if (cdcFile === null) {
+          errors.push("CDC file not uploaded");
+        }
+        var errorMessage = errors.join(" ") + ".";
```
Comment (waffy1901) [inline CDC-Data-Reconciliation-Frontend/src/components/CreateReport.jsx:113]: I believe setShowError(true); should be added after this line.

**Comment by john-baute** [comment]
@waffy1901 changes made

**Comment by john-baute** [comment]
New changes made @waffy1901

**Comment by waffy1901** [comment]
The "No attributes selected" error handling works great for the automatic report but it is not implemented for the manual report.

**Comment by john-baute** [comment]
Are attributes being passed in manual reports? I did not see in the code where that was being done for manual reports which was why I skipped it for that.

**Comment by john-baute** [comment]
On line 89 in automatic reports, the attributes are being passed into the form data, but there is nothing like that on line 138 for manual reports which is what is confusing me.

**Comment by waffy1901** [comment]
> On line 89 in automatic reports, the attributes are being passed into the form data, but there is nothing like that on line 138 for manual reports which is what is confusing me.

Good call out, I will have to fix that. You can leave the code as it is then.

**Review by waffy1901 — APPROVED** [review]
Great work!

**Pull request merged**